viewing_party/party.py

```diff
@@ -0,0 +1,135 @@
+def create_movie(title, genre, rating):
+    new_movie = {
+        "title" : title,
+        "genre" : genre,
+        "rating" : rating,
+    }
+
+    # != and "is not" do the same
+    if title != None and \
+        genre != None and \
+        rating != None:
+        return new_movie
+
+def add_to_watched(user_data, movie):
+    user_data["watched"] = [movie]
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"] = [movie]
+    return user_data
+
+def watch_movie(user_data, title):
+    for friend_watched in user_data["watchlist"]:
+        if friend_watched["title"] == title:
+            user_data["watchlist"].remove(friend_watched)
+            user_data["watched"].append(friend_watched)
+    return user_data
```

Comment:
I'm curious about the use of the naming friends_watched here.  For this particular function, we are just working with the user's data: 

`for movie in user_data["watchlist"]:
        if title == movie["title"]:
            user_data["watchlist"].remove(movie)
            user_data["watched"].append(movie)
    return user_data`